build.gradle

```diff
@@ -61,6 +61,7 @@ class OpenWhiskPlatform {
         The 'zipFileName' property is the root file name to use for archives.
      */
     static String zipFileName
+    static String packageVersion
```

Comment:
I think this belongs a project extension rather than an attribute of OpenWhiskPlatform.  Platform really is intended to deal solely with the process/os aspects of the build and is less volatile than version.

Additional thought on why this is important — I think it would be a Bad Idea (but possible and implicitly supported) to build different version numbers for different architectures if we set the `packageVersion` for each platform.

That means taking off the `OpenWhisk.` and `p.` below.

Reply:
Thx. Done.